### ISSUE TITLE
Edge of focus ring sometimes not visible in compact sp-action-group

### ISSUE THREAD

**Issue by davidclark87** [adobe/spectrum-web-components#1035]
### Expected Behaviour

The focus ring should completely wrap the element. See equivalent in [React Spectrum](https://react-spectrum.adobe.com/react-spectrum/ActionGroup.html#density):

![image](https://user-images.githubusercontent.com/15110225/101131499-3d166880-35ba-11eb-81e9-56b48513abed.png)

### Actual Behaviour

For Horizontal buttons, the right hand side of the focus ring is not visible for any but the rightmost button in the group:
![image](https://user-images.githubusercontent.com/15110225/101131416-1a844f80-35ba-11eb-9a28-47f6c2888707.png)

![image](https://user-images.githubusercontent.com/15110225/101131427-207a3080-35ba-11eb-864a-be77951894e0.png)

![image](https://user-images.githubusercontent.com/15110225/101131440-27a13e80-35ba-11eb-9917-f86960277af9.png)

For Vertical buttons, the bottom edge is not shown:
![image](https://user-images.githubusercontent.com/15110225/101131714-9f6f6900-35ba-11eb-8cef-33de2f9ca2e5.png)

#### Steps to Reproduce

1. Go to https://opensource.adobe.com/spectrum-web-components/components/action-group
2. Tab to the compact buttons in the Horizontal example

#### Platform and Version
Tested on Chrome 87 on Mac.

#### Sample Code that illustrates the problem
See docs

#### Logs taken while reproducing problem
N/A

**Comment by Westbrook** [comment]
This has been corrected in the Spectrum CSS release that just went out today. It'll take a couple of days before I can wade through all of the changes therein, but then this will be managed automatically when processing their output into our components.

**Comment by Westbrook** [comment]
Released
